receiver/k8sobjectsreceiver/receiver.go

```diff
@@ -50,17 +51,21 @@ func newReceiver(params receiver.Settings, config *Config, consumer consumer.Log
 		return nil, err
 	}
 
-	for _, object := range config.Objects {
-		object.exclude = make(map[apiWatch.EventType]bool)
-		for _, item := range object.ExcludeWatchType {
-			object.exclude[item] = true
+	objects := make([]*K8sObjectsConfig, len(config.Objects))
+	for i, obj := range config.Objects {
+		copied := *obj // Copy the object
```

Comment:
```suggestion
                // TODO: move mutable struct fields on their own struct out of Config struct
		copied := *obj // Copy the object
```

Reply:
Instead of TODOs in code lets make issues.